docs/_data/team.js

```diff
@@ -0,0 +1,150 @@
+export const core = [
+  {
+    avatar: 'https://www.github.com/yyx990803.png',
+    name: 'Evan You',
+    title: 'Creator',
+    org: 'Vue.js',
+    orgLink: 'https://vuejs.org/',
+    desc: 'Husband, father of two, independent open source developer.',
+    links: [
+      { icon: 'github', link: 'https://github.com/yyx990803' },
+      { icon: 'twitter', link: 'https://twitter.com/youyuxi' }
+    ],
+    sponsor: 'https://github.com/sponsors/yyx990803'
+  },
+  {
+    avatar: 'https://www.github.com/patak-dev.png',
+    name: 'Patak',
+    title: 'A collaborative being',
+    org: 'StackBlitz',
+    orgLink: 'https://stackblitz.com/',
+    desc: 'Core team member of Vite. Team member of Vue.',
+    links: [
+      { icon: 'github', link: 'https://github.com/patak-dev' },
+      { icon: 'twitter', link: 'https://twitter.com/patak_dev' }
+    ],
+    sponsor: 'https://github.com/sponsors/patak-dev'
+  },
+  {
+    avatar: 'https://www.github.com/antfu.png',
+    name: 'Anthony Fu',
+    title: 'A fanatical open sourceror',
+    org: 'NuxtLabs',
+    orgLink: 'https://nuxtlabs.com/',
+    desc: 'Core team member of Vite & Vue Working at NuxtLabs.',
```

Comment:
In all places we uses periods at the end of sentences.

```suggestion
    desc: 'Core team member of Vite & Vue. Working at NuxtLabs.',
```